.github/pull_request_template.md

```diff
@@ -2,6 +2,11 @@
 
  - [ ] Title of this PR explains the impact of the change.
  - [ ] The description provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
- - [ ] The CHANGELOG.md file in the *Unreleased* section has been updated to explain the change which will be included in the release notes.
- - [ ] If a component uses a new or changed internal endpoint of another component, this is mentioned in the CHANGELOG.md.
-- [ ] If this PR creates a new endpoint, or adds a new configuration flag, the endpoint / config-flag checklist (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
+ - [ ] If end-points have been added or changed: the endpoint / config-flag checklist (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
+ - [ ] If a schema migration has been added, I ran `make git-add-cassandra-schema` to update the cassandra schema documentation.
+ - Section *Unreleased* of CHANGELOG.md contains the following bits of information:
```

Comment:
this contains (in spirit, i hope) the first two items that were removed.

Reply:
```suggestion
 - [ ] Section *Unreleased* of CHANGELOG.md contains the following bits of information:
```

Reply:
Did you want this to just be a bullet point without a checkbox? It looks a bit weird when rendered.

Reply:
thanks.  :)

it's amazing how hard easy things are when you're tired and distracted.